File: ass-ade-v1.1/tests/fixtures/rebuilt-out/bridges/rust/src/a3_og_features/bridge_feature.rs
use std::path::PathBuf;

use crate::a0_qk_constants::bridge_contract::BridgeManifestSummary;
use crate::a2_mo_composites::python_bridge_client::PythonBridgeClient;

pub fn describe_bridge(start: PathBuf) -> Result<BridgeManifestSummary, String> {
    PythonBridgeClient::new(start)?.manifest()
}

File: tests/v11/fixtures/rebuilt-out/bridges/typescript/src/a3_og_features/bridge_feature.ts
import type { BridgeManifest } from "../a0_qk_constants/bridge_contract.js";
import { PythonBridgeClient } from "../a2_mo_composites/python_bridge_client.js";

export function describeBridge(root: string = process.cwd()): BridgeManifest {
  return new PythonBridgeClient(root).manifest();
}
